.azure-pipelines/buildAndPackage.yml

```diff
@@ -30,10 +30,37 @@ pr: none
 pool:
   vmImage: windows-latest
```

Comment:
also last time I checked, some of the tasks we were using were not compatible with anything else than windows. But it might be worth trying ubuntu latest again

Reply:
I noticed that using windows-latest required a few other tasks, such as downloading Java. I'm not sure that Ubuntu requires that which is why I have it as the vmImage for the main SDK and Beta library, though I have left it on this pipeline as it currently seems to function.

Reply:
I always add tasks to setup the required "tools". Worst case it's missing and gets installed, best case it's already here and it's a no-op. If the setup tasks are missing and the vm image gets updated not to include the tool anymore, the pipeline will start failing. It also allows to get the specific version that's required.

Reply:
The virtual environments for the DevOp images can be found here:

https://github.com/actions/virtual-environments/tree/main/images

The readme in each folder describes what software is on the image by default. I think Java is available on both Ubuntu and Windows.

Reply:
I've had the main version of node or the installed versions of node pulled from under my feet due to an image refresh when I was working on SPFx (which is known to lag in terms of which node version it supports) that I don't trust the pre-installed stuff anymore 😅
again, always add the install task, best case it's a no-op, worst case it'll install something, making your runtime longer but at least the pipeline will keep working.

Reply:
Sounds good, I believe these steps are currently on the core pipeline already, will add them to the beta and main library pipelines.